lib/AST/CASTBridging.cpp

```diff
@@ -651,14 +651,19 @@ void Plugin_unlock(PluginHandle handle) {
 bool Plugin_sendMessage(PluginHandle handle, const BridgedData data) {
   auto *plugin = static_cast<LoadedExecutablePlugin *>(handle);
   StringRef message(data.baseAddress, data.size);
-  return plugin->sendMessage(message);
+  return bool(plugin->sendMessage(message));
```

Comment:
I'm fairly sure this will still fail when there's an error. LLVM has a `consumeError` to just eat it, though that makes me wonder if we should be passing in an error string to set.